website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,243 @@
+- When displaying hierarchical data that allows users to expand rows to reveal related data without leaving the table.
+- When advanced keyboard navigation is required, such as moving between cells, focusing and interacting with expandable rows, and managing nested structures.
+- When implementing complex interactions like multi-select, nested rows, or expendable content, which are not supported in the standard Table.
+
+### When not to use
+
+- If sorting and pagination are sufficient for your dataset, use the standard Table instead.
+- When displaying static or minimal data that doesn’t require advanced interactivity.
+- When a visual representation like charts or graphs better conveys the data.
+- As a layout mechanism for structuring content that isn’t tabular data.
+- When replicating spreadsheet-like functionality with extensive in-cell editing or calculations.
+
+## Columns
+
+### Sorting
+
+![Header column sorting](/assets/components/table/advanced-table/table-sorting.png)
+
+- Sorting is not relevant for all content, so consider when to apply sorting thoughtfully.
+- Columns with sorting enabled are interactive and include hover, active, and focus states.
+- An Advanced Table may only be sorted by a single column at a time.
+
+
+### Tooltips
+
+Labels within the Advanced Table column should be clear, concise, and straightforward. In the case that more context or details are necessary, a [Tooltip](/components/tooltip) can be used in conjunction with the label but should be used sparingly and as a last resort.
+
+![Tooltips in a Header Column](/assets/components/table/advanced-table/table-tooltip-example.png)
+
+Some examples where it may be useful to include additional context in a tooltip include:
+- When the label contains a product or HashiCorp-specific term.
+- When the label refers to a setting that can be changed elsewhere in the application.
+
+
+## Width 
+
+Column width is determined by manually resizing the header column and cells within Figma. As a best practice, column width should be adjusted to fit the longest data type within the cell.
+
+## Placement
+
+!!! Info
+
+The column placement property is only relevant within Figma and doesn’t exist as a property in code.
+!!!
+
+Column placement determines the visual styling based on where the column is placed relative to other columns in the Advanced Table.
+
+![Advanced Table column placement example](/assets/components/table/advanced-table/table-col-placement.png)
+
+## Alignment
+
+The alignment of text and content within an Advanced Table impacts the readability and speed at which users can effectively parse the information. The chosen alignment method depends on the content within the cell, and relative position within the advanced table.
+
+!!! Info 
+Helios uses left and right for alignment to match the Ember component API. This documentation follows that convention for consistency across design and development.
```

Comment:
[small nit]

```suggestion
Helios uses left and right for alignment to match the Ember component API. This documentation follows that convention for consistency across design and code.
```

Reply:
Also, all of the content following this uses `start` and `end` alignment, which is contrary to this note. I'll try to catch them where I can, but this probably also needs to be updated in the standard table docs. I thought we had a task to track that or had already done it, but LMK if you want me to create one or if you want to handle it in this PR.